src/views/EdgeApplications/FormFields/FormFieldsCreateEdgeApplications.vue

```diff
@@ -204,6 +208,7 @@
 <template>
   <FormHorizontal
     title="General"
+    :isDrawer="sisDrawer"
```

Comment:
passou um "s" aqui e nas outras
```suggestion
    :isDrawer="isDrawer"
```

Reply:
typo resolved